proto/steward/v4/cellar_v2.proto

```diff
@@ -46,37 +46,28 @@ import "curve.proto";
 */
 message CellarV2 {
     // The function you wish to execute on the target cellar
-    oneof function {
-        // Represents function `addPosition(uint256 index, address position)`
-        AddPosition add_position = 1;
+    oneof function { 
         // Represents function `callOnAdaptor(AdaptorCall[] memory data)`
-        CallOnAdaptor call_on_adaptor = 2;
+        CallOnAdaptor call_on_adaptor = 1;
         // Represents function `removePosition(uint256 index)`
-        RemovePosition remove_position = 3;
+        RemovePosition remove_position = 2;
         // Represents function `setHoldingPosition(uint32 position_id)`
-        SetHoldingPosition set_holding_position = 4;
+        SetHoldingPosition set_holding_position = 3;
         // Represents function `setStrategistPayoutAddress(address payout)`
-        SetStrategistPayoutAddress set_strategist_payout_address = 5;
+        SetStrategistPayoutAddress set_strategist_payout_address = 4;
         // Represents function `swapPositions(uint256 index1, uint256 index2)`
-        SwapPositions swap_positions = 6;
+        SwapPositions swap_positions = 5;
         // Represents function `setShareLockPeriod(uint256 newLock)`
-        SetShareLockPeriod set_share_lock_period = 7;
-    }
+        SetShareLockPeriod set_share_lock_period = 6;
```

Comment:
I think we're missing some functions that we added on in 3.x-main as SP functions

`SetRebalanceDeviation`
`SetPlatformFee`
`InitiateShutdown`
`LiftShutdown`